src/frontend/Komponenter/Behandling/Simulering/Tilbakekreving.tsx

```diff
@@ -26,6 +27,8 @@ export interface ITilbakekreving {
 export const TilbakekrevingsvalgTilTekst: Record<ITilbakekrevingsvalg, string> = {
     OPPRETT_MED_VARSEL: 'Opprett med varsel',
     OPPRETT_UTEN_VARSEL: 'Opprett uten varsel',
+    OPPRETT_AUTOMATISK:
+        'Opprett automatisk behandling av tilbakekreving under 4 ganger rettsgebyret',
```

Comment:
Skulle vi slengt på `(ingen tilbakekreving)` i parentes her - eller blir det voldsomt mye tekst?

Reply:
Burde gå greit. La ved screenshot for å vise hvordan det blir.